src/database/schema/ChatSchema.ts

```diff
@@ -84,7 +89,8 @@ class ChatSchema implements BaseDbSchema {
         this.status,
         this.createdAt,
         this.updatedAt,
-        this.rawJson
+        this.rawJson,
+        this.attachmentJson
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `attachmentJson` property is inserted into the database without any validation or transformation. It would be safer to ensure that `attachmentJson` is always stored as a stringified JSON object.
```diff
-        this.rawJson,
-        this.attachmentJson
+        this.rawJson,
+        JSON.stringify(this.attachmentJson)
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->